runtime/doc/dev_tools.txt

```diff
@@ -88,8 +88,6 @@ Then, in another terminal:
     gdb build/bin/nvim
     target remote localhost:7777
 <
-- See also test/functional/testutil.lua https://github.com/neovim/neovim/blob/3098b18a2b63a841351f6d5e3697cb69db3035ef/test/functional/testutil.lua#L38-L44.
```

Comment:
the URL was broken but we could fix it: https://github.com/neovim/neovim/blob/a4fc3bb0e68c8b078377fd9826e4cca3b4b3fdbf/test/functional/testutil.lua#L29-L51

or mention "see nvim_argv in ..."

Reply:
We haven't merged this PR yet, how could we refer to a commit hash that hasn't been created?

Reply:
I'll go ahead with the "see nvim_argv in ..." option